.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-ibm-ocp-ai
+    kind: solution
+    catalog_id: f64499c8-eb50-4985-bf91-29f9e605a433
+    offering_id: e221fc14-094f-4b1e-ab21-d884378230e7
+    variations:
+      - name: fully-configurable
+        mark_ready: true
+        install_type: fullstack
+        scc:
+          instance_id: 1c7d5f78-9262-44c3-b779-b28fe4d88c37
+          region: us-south
+          scope_resource_group_var_name: existing_resource_group_name
```

Comment:
The whole SCC section is not needed here - we don't claim any SCC rules for this DA, and we never will since its just an entrypoint to OCP DA